// function tryAndCatch(tryFun, errorFun, finalFun) {
//     try {
//         tryFun();
//     } catch (error) {
//         errorFun();
//     } finally {
//         finalFun();
//     }
// }
